builtin/providers/aws/resource_aws_iam_group_policy_attachment.go

```diff
@@ -0,0 +1,174 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsIamGroupPolicyAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsIamGroupPolicyAttachmentCreate,
+		Read:   resourceAwsIamGroupPolicyAttachmentRead,
+		Update: resourceAwsIamGroupPolicyAttachmentUpdate,
+		Delete: resourceAwsIamGroupPolicyAttachmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"group": &schema.Schema{
```

Comment:
Should this be a `ForceNew` attribute? Looking at `resourceAwsIamGroupPolicyAttachmentUpdate` it doesn't seem to detect and act on a change of `group`